aspnetcore/blazor/webassembly-native-dependencies.md

```diff
@@ -0,0 +1,159 @@
+---
+title: ASP.NET Core Blazor WebAssembly native dependencies
+author: guardrex
+description: Learn how to build Blazor WebAssembly apps with native dependencies built to run on WebAssembly in the browser.
+monikerRange: '>= aspnetcore-6.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 10/28/2021
+no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: blazor/webassembly-native-dependencies
+---
+# ASP.NET Core Blazor WebAssembly native dependencies
+
+*This article applies to Blazor WebAssembly.*
+
+Blazor WebAssembly apps can use native dependencies built to run on WebAssembly. You can statically link native dependencies into the .NET WebAssembly runtime using the .NET WebAssembly build tools, the same tools used to [ahead-of-time (AOT) compile](xref:blazor/host-and-deploy/webassembly#ahead-of-time-aot-compilation) a Blazor app to WebAssembly and to [relink the runtime to remove unused features](xref:blazor/host-and-deploy/webassembly#runtime-relinking).
+
+## .NET WebAssembly build tools
+
+The .NET WebAssembly build tools are based on [Emscripten](https://emscripten.org/), a compiler toolchain for the web platform. To install the .NET WebAssembly build tools, use either of the following approaches:
+
+* Select the optional component in the Visual Studio installer.
```

Comment:
Could we be more explicit about its name here? Is it called ".NET WebAssembly build tools" in VS?

Reply:
I'll check on the name. I've removed VS 2019 but haven't installed VS 2022 yet ... I'll do that today.